pkg/statistics/hot_peer_cache.go

```diff
@@ -153,6 +155,37 @@ func (f *HotPeerCache) CollectExpiredItems(region *core.RegionInfo) []*HotPeerSt
 	return items
 }
 
+// CheckLeaderChange checks whether the leader of the region has changed.
+func (f *HotPeerCache) CheckLeaderChange(region *core.RegionInfo) []*HotPeerStat {
```

Comment:
Do we need to add a unit for it?

Reply:
I think it has been added in [hot_peer_cache_test.go](https://github.com/tikv/pd/pull/9372/files#diff-18a77c47ea41755636f2b09a6b8cb7fd9624586d42e7180335ba0507254fa3f0)